apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -52,53 +54,101 @@ export default function UserThreadPanel({
+
   const loadMore = async () => {
     console.log('🟡 Loading More Content')
     setLoading(true)
+    let moreThreads: Thread[] = []
 
-    const moreThreads = await getThreads({
+    if(page === 'profile' && !session?.user) {
+      moreThreads = await fetchBrowseThreads();
+    }else{
+     moreThreads = await getThreads({
       jwt: session!.user?.hasuraJwt,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)